### PR TITLE
[REACH-27771] Increase constant MAX_COMPATIBILITY_VERSION to be 7. Co…

### PR DESCRIPTION
**Pull request by mark-langbehn**
…mment out testcase that was failing.

### CODE REVIEW

File: src/main/java/com/iheartradio/m3u8/Constants.java [hunk 1]
```diff
@@ -111,7 +111,7 @@
 
     public static final int[] UTF_8_BOM_BYTES = {0xEF, 0xBB, 0xBF};
     public static final char UNICODE_BOM = '\uFEFF';
-    public static final int MAX_COMPATIBILITY_VERSION = 5;
+    public static final int MAX_COMPATIBILITY_VERSION = 7;    // upstream version 0.2.7 of this m3u8 code has EXT_X_VERSION:5
```
Comment (joshuapaulallen) [inline src/main/java/com/iheartradio/m3u8/Constants.java:114]: Would it be worthwhile to add a test case that verifies parsing an .m3u8 file using this version works as expected?

**Review by MasonSteele — APPROVED** [review]

File: src/main/java/com/iheartradio/m3u8/Constants.java (hunk 1 above)
Comment (mark-langbehn) [reply src/main/java/com/iheartradio/m3u8/Constants.java:114]: done.  added m3u8 sample files, and unit-tests in MediaPlaylistParserStartTimeTest.java

Comment (mark-langbehn) [reply src/main/java/com/iheartradio/m3u8/Constants.java]: done. added testcase in MediaPlaylistParserStartTimeTest.java

**Review by joshuapaulallen — APPROVED** [review]
Good deal.  Ready to merge.

Will there also be a reach-engine PR to use this forked version of open-m3u8?

**Comment by mark-langbehn** [comment]
Added a section to the top of the README.md based on feedback from James.

**Comment by leafknode** [comment]
looks ok to me but I'll defer approval to @joshuapaulallen

**Comment by joshuapaulallen** [comment]
Looks good to me as well.  Once we get write permissions to this repo, we will merge.

**Pull request merged**